recipe/patches/0002-Redis-deps-now-build-but-do-not-link.patch

```diff
@@ -3,7 +3,8 @@ From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
 Date: Fri, 3 Dec 2021 10:46:36 -0800
 Subject: [PATCH 2/6] Redis deps now build but do not link
 
-Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
```

Comment:
Given you've contributed something to this patch, too, I think you should also be present in the commit message.